controllers/openstackcluster_controller.go

```diff
@@ -610,7 +610,7 @@ func (r *OpenStackClusterReconciler) SetupWithManager(ctx context.Context, mgr c
 				}
 
 				if annotations.IsExternallyManaged(c) {
-					log.V(4).Info("OpenStackCluster is externally managed, skipping mapping.")
+					log.V(4).Info("OpenStackCluster was externally managed, skipped mapping")
```

Comment:
I would leave this as present tense, as it describes the current, ongoing state of the object.